core/integration/container_test.go

```diff
@@ -78,7 +79,9 @@ func TestContainerFrom(t *testing.T) {
 			}
 		}`, &res, nil)
 	require.NoError(t, err)
-	require.Equal(t, res.Container.From.File.Contents, "3.18.2\n")
+
+	releaseStr := res.Container.From.File.Contents
+	require.True(t, strings.HasPrefix(releaseStr, distconsts.AlpineVersion), releaseStr)
```

Comment:
super nitpick: this will give a failure like `false is not true` which isn't very helpful, might be better to assert against `distconsts.AlpineVersion+"\n"`

not blocking, since these tests _probably_ won't start failing anyway

Reply:
`+1`

Reply:
:tada: Fixed this up now!